### ISSUE TITLE
Datepicker does not scroll

### ISSUE THREAD

**Issue by nathanmerrill** [magento/magento2#7903]
### Preconditions

On Magento 2.1.1
Chrome on Linux Mint

### Steps to reproduce

I'm using the tier pricing, but I imagine this occurs in other locations on the admin side.

1. Create the following class

```

use Magento\Framework\Stdlib\ArrayManager;
use Magento\Ui\DataProvider\Modifier\ModifierInterface;
class DateRangeTier implements ModifierInterface
{
    protected $arrayManager;
    
    public function __construct(ArrayManager $arrayManager) {
        $this->arrayManager = $arrayManager;
    }
    
    public function modifyMeta(array $meta)
    {
        $tierPricePath = $this->arrayManager->findPath('tier_price', $meta, null, 'children');
        if ($tierPricePath) {
            $meta = $this->arrayManager->merge(
                $tierPricePath,
                $meta,
                $this->getTierPriceStructure()
            );
        }
        return $meta;
    }
    
    public function modifyData(array $data){return $data;}
    
    protected function getTierPriceStructure()
    {
        return [
            'children' => [
                'record' => [
                    'children' => [
                        'start_date' => [
                            'arguments' => [
                                'data' => [
                                    'config' => [
                                        'dataType' => 'date,
                                        'formElement' => 'date',
                                        'componentType' => 'field',
                                        'label' => __('Start Date'),
                                        'dataScope' => 'start_date',
                                    ],
                                ],
                            ],
                        ],
                        'end_date' => [
                            'arguments' => [
                                'data' => [
                                    'config' => [
                                        'dataType' => 'date',
                                        'formElement' => 'date',
                                        'componentType' => 'field',
                                        'label' => __('End Date'),
                                        'dataScope' => 'end_date',
                                    ],
                                ],
                            ],
                        ],
                    ],
                ],
            ],
        ];
    }

```
2. Add the class to your `etc/adminhtml/di.xml`:

```
<virtualType name="Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Pool" type="Magento\Ui\DataProvider\Modifier\Pool">
        <arguments>
            <argument name="modifiers" xsi:type="array">
                <item name="subscriptionPricing" xsi:type="array">
                    <item name="class" xsi:type="string">Path\To\DateRangeTier</item>
                    <item name="sortOrder" xsi:type="number">95</item>
                </item>
            </argument>
        </arguments>
    </virtualType>
```
3. Edit a product, and select Advanced Pricing.  Add a Tier, and scroll right.

### Expected result
The datepicker should scroll with the fields.

### Actual result
The datepicker does not scroll with the rest of the fields.  (I have more fields than the ones listed above, and so they cover up other fields).  

Screenshot:
![screenshot_2016-12-20_10-41-27](https://cloud.githubusercontent.com/assets/2955071/21361376/0102a8c2-c6a1-11e6-9c96-9d53f4826122.png)

**Comment by nathanmerrill** [comment]
A potential CSS fix is:

```
tr .admin__field-control {
    position: relative;
}
```

**Comment by magento-engcom-team** [comment]
@nathanmerrill, thank you for your report.
We've created internal ticket(s) MAGETWO-81917 to track progress on the issue.

**Comment by lionelalvarez** [comment]
I am working on it at #mm17es

**Comment by okorshenko** [comment]
The issue has been fixed and delivered to 2.1-develop branch. Will be available with next 2.1 release

**Comment by sidolov** [comment]
Hi @nathanmerrill. Thank you for your report. 
The issue has been fixed in magento/magento2#16775 by @hitesh-wagento in 2.2-develop branch
Related commit(s):
 - [7180790d2fc10cc55efe4aa4499c14b8d5dc360a](https://github.com/magento/magento2/commit/7180790d2fc10cc55efe4aa4499c14b8d5dc360a)

The fix will be available with the upcoming 2.2.7 release.
